### PR TITLE
fix: CQDG-415 fix variant template map and add study_code

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

File: index-task/src/main/resources/templates/template_variant_centric.json [hunk 1]
```diff
@@ -793,19 +805,23 @@
               "type": "keyword",
               "index": false
             },
-            "transmissions": {
+            "transmission": {
+              "type": "keyword",
+              "index": false
```
Comment (jecos) [inline index-task/src/main/resources/templates/template_variant_centric.json:810]: remove

File: index-task/src/main/resources/templates/template_variant_centric.json [hunk 2]
```diff
@@ -793,19 +805,23 @@
...
+            },
+            "zygosity": {
               "type": "keyword",
               "index": false
```
Comment (jecos) [inline index-task/src/main/resources/templates/template_variant_centric.json:814]: remove

**Review by jecos — CHANGES_REQUESTED** [review]

File: index-task/src/main/resources/templates/template_variant_centric.json (hunk 2 above)
Comment (adipaul1981) [reply index-task/src/main/resources/templates/template_variant_centric.json]: done

File: index-task/src/main/resources/templates/template_variant_centric.json (hunk 1 above)
Comment (adipaul1981) [reply index-task/src/main/resources/templates/template_variant_centric.json]: done

**Pull request merged**